rootex/framework/systems/render_system.cpp

```diff
@@ -12,6 +12,7 @@ RenderSystem* RenderSystem::GetSingleton()
 RenderSystem::RenderSystem()
     : m_Renderer(new Renderer())
     , m_Camera(new CameraVisualComponent())
+    , m_DefaultCamera(m_Camera)
```

Comment:
Initialization in initialization lists happens in the opposite order of the way member data is specified in the class definition. This way, m_DefaultCamera will get nullptr as its value